rrfs-test/validated_yamls/templates/obtype_config/msonet_specificHumidity_188.yaml

```diff
@@ -40,10 +40,29 @@
 
        obs filters:
          # ------------------
-         # specificHumidity
+         # specificHumidity (188)
          # ------------------
+         # Reject all obs with QualityMarker > 3
+         - filter: RejectList
+           apply at iterations: 0,1
+           where:
+           - variable: ObsType/specificHumidity
+             is_in: 188
+           - variable: QualityMarker/specificHumidity
+             is_in: 4-15
+
+         # Time window filter
+         - filter: Domain Check
+           apply at iterations: 0,1
+           where:
+             - variable:
+                 name: MetaData/timeOffset # units: s
+               minvalue: -360
+               maxvalue:  360
+           action:
+             name: reduce obs space
```

Comment:
As discussed today, could we remove this until #209 is merged? Using `reduce obs space` breaks GETKF until OOPS is updated.

Reply:
Yes, I can remove it in this PR. I think you'll still probably have conflicts when you add it back in because I reordered a bunch of the filters.

Reply:
Yeah - I'll probably have a bit to work through with my PR after this one is merged hah.